taxi/views.py

```diff
@@ -11,8 +11,11 @@ def index(request):
     num_drivers = Driver.objects.count()
     num_cars = Car.objects.count()
     num_manufacturers = Manufacturer.objects.count()
+    num_visits = request.session.get("num_visits", 0)
```

Comment:
You forgot about @login_required to index

Reply:
I didn't forget, I didn't put it there on purpose. As of me, it doesn't make sense to use @login_required there

Reply:
Okay, but according to task it should be there